packages/transaction-controller/CHANGELOG.md

```diff
@@ -6,6 +6,30 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [6.0.0]
+### Added
+- Update transaction controller to automatically initiate, finalize, and cancel approval requests as transactions move through states ([#1241](https://github.com/MetaMask/core/pull/1241))
+  - The `ApprovalController:addRequest` action will be called when a new transaction is initiated
+  - The `ApprovalController:rejectRequest` action will be called if a transaction fails
+  - The `ApprovalController:acceptRequest` action will be called when a transaction is approved 
+
+### Changed
+- **BREAKING:** Bump to Node 16 ([#1262](https://github.com/MetaMask/core/pull/1262))
+- **BREAKING:** Update `@metamask/network-controller` dependency ([#1367](https://github.com/MetaMask/core/pull/1367))
+  - This affects the `getNetworkState` and `onNetworkStateChange` constructor parameters
+- **BREAKING:** Change format of chain ID in state to `Hex` ([#1367](https://github.com/MetaMask/core/pull/1367))
+  - The `chainId` property of the `Transaction` type has been changed from `number` to `Hex`
+  - The `chainId` property of the `TransactionMeta` type has been changed from a decimal `string` to `Hex`, and the `transaction` property has been updated along with the `Transaction` type (as described above).
+  - The state property `transactions` is an array of `TransactionMeta` objects, so it has changed according to the description above.
+    - This requires a state migration: each entry should have the `chainId` property converted from a decimal `string` to `Hex`, and the `transaction.chainId` property changed from `number` to `Hex`.
+  - The `addTransaction` and `estimateGas` methods now expect the first parameter (`transaction`) to use type `Hex` for the `chainId` property.
+  - The `updateTransaction` method now expects the `transactionMeta` parameter to use type `Hex` for the `chainId` property (and for the nested `transaction.chainId` property)
+- **BREAKING:** Add `messenger` as required constructor parameter ([#1241](https://github.com/MetaMask/core/pull/1241))
+- Add `@metamask/utils` dependency ([#1367](https://github.com/MetaMask/core/pull/1367))
```

Comment:
Should we also add that `@metamask/approval-controller` is a new dependency?

Reply:
Yes, I think so. Done in https://github.com/MetaMask/core/pull/1385/commits/a5bbd82ea57e57c1c414824eedbe6331dc967e97